model/trace/database.yaml

```diff
@@ -453,6 +454,28 @@ groups:
         examples: [ '"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"' ]
       - ref: server.address
       - ref: server.port
+      - id: cluster_name
+        type: string
+        requirement_level:
+          recommended: >
+            Should be collected from the relevant HTTP response header if available.
+        tag: call-level-tech-specific
+        brief: >
+          Represents the identifier of an Elasticsearch cluster.
+        examples: ["e9106fc68e3044f0b1475b04bf4ffd5f"]
+        note: >
+          For Elastic Cloud hosted deployents the "X-Found-Handling-Cluster" HTTP response header contains the cluster ID.
```

Comment:
Any recommendation for non-cloud hosted deployments? Is such attribute available only on Cloud hosted?

Reply:
This is currently only available on cloud-hosted deployments, but there have been some discussions about the feasibility and usefulness of adding this to self-managed installations in the future.

Reply:
I've updated the recommendation to mention that this applies to cloud only (for now).